docker/requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+# static linters
+black==20.8b1
+isort==5.7.0
```

Comment:
```suggestion
black==21.10b0
isort==5.10.0
```

Latest release as of today

Feel free to disregard.  Making the python tidy is a very welcome thing :)

Reply:
Sure, I need some time though

Reply:
@ioggstream take your time. I'm just a passerby. (I don't want to disrupt) 😊